Formula/lua.rb

```diff
@@ -93,68 +92,68 @@ def caveats
   end
 
   test do
-    system "#{bin}/lua", "-e", "print ('Ducks are cool')"
+    assert_match "Homebrew is awesome!", shell_output("#{bin}/lua -e \"print ('Homebrew is awesome!')\"")
   end
 end
 
 __END__
```

Comment:
It seems like this patch will be around for a while, is it submitted upstream already? And/or can we migrate it to Homebrew/formula-patches?

Reply:
This patch has been around for quite a while: https://github.com/Homebrew/legacy-homebrew/pull/5043

I think it's somewhat unique to the way Homebrew packages things? Though I don't see how it could hurt upstream, and probably only helps.

I don't mind migrating it to Homebrew/formula-patches. Do you mind if I pursue that in a separate PR, if (fingers crossed) this one gets merged?

Reply:
I only just realised exactly _how old_ this patch is. It's really alarming.

I'm going to try to see if I can track down someone familiar with lua internals who can have a look at the patch.